caikit/runtime/servicers/global_predict_servicer.py

```diff
@@ -329,7 +329,7 @@ def _handle_predict_exceptions(self, model_id: str, request_name: str):
             ).inc()
             raise CaikitRuntimeException(
                 StatusCode.INVALID_ARGUMENT,
-                f"Exception raised during inference. This may be a problem with your input: {e}",
+                f"{e}",
```

Comment:
don't really need the f-string?

Reply:
It seems we do, as removing it causes "Internal server error" to display instead of the exception